pkg/controller/controllers.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 // AddSpecSyncers adds the controllers that get updates from transport layer and apply/delete CRs to the Manager.
-func AddSpecSyncers(mgr ctrl.Manager, bundleUpdatesChan chan *bundle.ObjectsBundle) error {
-	err := bundles.AddLeafHubBundlesSpecSync(ctrl.Log.WithName("bundles-syncer"), mgr, bundleUpdatesChan)
+func AddSpecSyncers(mgr ctrl.Manager, bundleUpdatesChan chan *bundle.ObjectsBundle, numOfClients int) error {
+	err := bundles.AddLeafHubBundlesSpecSync(ctrl.Log.WithName("bundles-syncer"), mgr, bundleUpdatesChan, numOfClients)
 	if err != nil {
```

Comment:
The if statement should be combined with the line above for consistency and good practice.

Reply:
done